source/_docs/tools/quick-bar.markdown

```diff
@@ -14,55 +14,45 @@ It can be launched in the frontend using [hotkeys](#hotkeys).
 
 ## Hotkeys
 
-Type these from anywhere in the application to launch the dialog.
+Type these from anywhere (see Notes) in the application to launch the dialog.
 
-| Mode  | Hotkey (Windows) | Hotkey (macOS)
-| ------------- | ------------- |------------- |
-| Entity Filter | Ctrl+P | ⌘+P |
-| Command | Ctrl+Shift+P | ⌘+Shift+P |
+| Mode  | Hotkey | Switch Modes
+| ------------- | ------------- | ------------- |
+| Entity Filter | `e` | Type `>` at start of input to switch to command palette.
+| Command Palette| `c` | Remove `>` from start of input to switch to entity filter.
 
 <div class='note'>
-  The application must have "focus" for the hotkey to register. If the dialog doesn't launch, try clicking in an empty area of the application and typing it again.
+  The application must have focus for the hotkey to register. If the dialog doesn't launch, try clicking into an empty part of the main content area of Home Assistant and type it again.
 </div>
 
-## Switching Modes
-
-When the dialog has already been launched, you can switch to other modes without losing your filter text.
-
-<p class='img'>
-  <img src='/images/docs/quick-bar/quick-bar-switch-modes.gif' alt='Quick Bar'>
-  Switch modes without losing filter text
-</p>
-
-| Current Mode | Action | Switches to Mode |
-| ------------- | ------------- | ------------- |
-| Entity Filter | Use [hotkey](#hotkeys) (or prepend ">" to input field) | Command
-| Command | Use [hotkey](#hotkeys) (or remove ">" from input field) | Entity Filter
+<div class='note'>
+  There is a <a href="https://github.com/home-assistant/frontend/issues/7442">known issue</a> right now where clicking on any links in sidebar will break the shortcuts. For now, you can either click anywhere in the main content area, or type "Esc" or "Shift+Tab" to remove focus from the sidebar. Then shortcuts should work again.
+</div>
 
-## Entity Filter Mode
+## Entity Filter
 
-*Hotkey: Ctrl+P (Windows) or ⌘+P (macOS)*
+*Hotkey: `e`*
 
-Similar to Developer Tools -> States view, but more lightweight and accessible from anywhere in the frontend.
+Similar to Developer Tools -> States, but more lightweight and accessible from anywhere in the frontend.
```

Comment:
I would say similar to Configuration -> Entities

Reply:
Done